pandablocks/utils.py

```diff
@@ -23,6 +25,9 @@ def words_to_table(
             expected to be the string representation of a uint32.
         table_fields_info: The info for tables, containing the number of words per row,
             and the bit information for fields.
+        convert_enum_indices: If True, converts enum indices to labels, the packed
+            value will be a list of strings. If False the packed value will be a
+            numpy array of the indices the labels correspond to.
```

Comment:
Shouldn't both instances of "packed" here be "unpacked"? 

I'd also prefer the first line to read more like `If True, convert all enum values to their string representation. Otherwise return enums as integer values`, but that's a minor preference.